.github/ISSUE_TEMPLATE/ppl_bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 🐛 PPL Bug Report
+about: Report a bug or unexpected behavior in PPL
+title: '[BUG] Brief description of the issue'
+labels: 'bug, untriaged, PPL'
+assignees: ''
+---
+**PPL Command/Query:**
+```ppl
+Paste your PPL command or query here
+```
+
+## Bug Description
+**Describe the bug**
+A clear and concise description of what the bug is and what went wrong.
+
+## Reproduction Steps
+**Steps to reproduce the behavior:**
```

Comment:
Ask for index mapping and sample data explicitly.

Reply:
agree, and maybe add another reminder for issue filler to sanitize the sensitive info/fields first.

Reply:
Both comments have been addressed